examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,646 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      ">> head data/kinglear.txt\n",
+      "\tKING LEAR\r\n",
+      "\r\n",
+      "\r\n",
+      "\tDRAMATIS PERSONAE\r\n",
+      "\r\n",
+      "\r\n",
+      "LEAR\tking of Britain  (KING LEAR:)\r\n",
+      "\r\n",
+      "KING OF FRANCE:\r\n",
+      "\r\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "run('head data/kinglear.txt')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This is just a `txt` file - it contains lines of text.\n",
+    "Let's take a look at the other dataset."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 136,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      ">> head data/SMSSpamCollection.csv\n",
+      "ham\tGo until jurong point, crazy.. Available only in bugis n great world la e buffet... Cine there got amore wat...\r\n",
+      "ham\tOk lar... Joking wif u oni...\r\n",
+      "spam\tFree entry in 2 a wkly comp to win FA Cup final tkts 21st May 2005. Text FA to 87121 to receive entry question(std txt rate)T&C's apply 08452810075over18's\r\n",
+      "ham\tU dun say so early hor... U c already then say...\r\n",
+      "ham\tNah I don't think he goes to usf, he lives around here though\r\n",
+      "spam\tFreeMsg Hey there darling it's been 3 week's now and no word back! I'd like some fun you up for it still? Tb ok! XxX std chgs to send, £1.50 to rcv\r\n",
+      "ham\tEven my brother is not like to speak with me. They treat me like aids patent.\r\n",
+      "ham\tAs per your request 'Melle Melle (Oru Minnaminunginte Nurungu Vettam)' has been set as your callertune for all Callers. Press *9 to copy your friends Callertune\r\n",
+      "spam\tWINNER!! As a valued network customer you have been selected to receivea £900 prize reward! To claim call 09061701461. Claim code KL341. Valid 12 hours only.\r\n",
+      "spam\tHad your mobile 11 months or more? U R entitled to Update to the latest colour mobiles with camera for Free! Call The Mobile Update Co FREE on 08002986030\r\n",
+      "\n",
+      ">> wc -l data/SMSSpamCollection.csv\n",
+      "5574 data/SMSSpamCollection.csv\r\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "run('head data/SMSSpamCollection.csv')\n",
+    "run('wc -l data/SMSSpamCollection.csv')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This dataset is a `csv` file with 5,574 rows and 2 columns recording the following attributes separated by a tab sign:\n",
+    "1. `Column 1`: The label (either `ham` or `spam`)\n",
+    "2. `Column 2`: The SMS as raw text (type `string`)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Example 1: word count\n",
```

Comment:
Here it may be good to do really basic syntax introduction, similar to "structure" section in https://s.apache.org/beam-yaml-pipelines-syntax

Reply:
That's a valid point, thanks

Reply:
Added